File: SEM 4/ADS Lab/CLASS 2 16-03-2022/calculator.cpp
#include <iostream>
#include <string.h>
#include <math.h>
using namespace std;


class calculator
{
   public:
       int a,b;

       void get()
       {
           cout<<"Enter the numbers = ";
           cin>>a>>b;
       }

        void sum();
        void subtract();
        void multiply();
        void divide();
        void power();
};


void calculator::sum()
{
    cout<<"Sum = "<<a+b;
}

void calculator::subtract()
{
    cout<<"Subtract = "<<a-b;
}

void calculator::multiply()
{
    cout<<"Product = "<<a*b;
}

void calculator::divide()
{
    cout<<"Division = "<<a/b;
}

void calculator::power()
{
    cout<<a<<" to the power of "<<b<<" is "<<pow(a,b);
}


int main()
{
    cout<<"1.ADD\n2.SUBTRACT\n3.MULTIPLICATION\n4.DIVISION\n5.POWER"<<endl;


    while(1)
    {
        calculator usr;

        //users



        int choice,a,b;
        cout<<"\nEnter the choice = ";
        cin>>choice;
        if(choice==0) break;


        //getting the input
        usr.get();

        switch(choice)
        {
            case 1:
                usr.sum();
                break;
            case 2:
                usr.subtract();
                break;
            case 3:
                usr.multiply();
                break;
            case 4:
                usr.divide();
                break;
            case 5:
                usr.power();
                break;
            default:
                cout<<"Wrong choice";
                break;
        }
    }

    return 1;
}

File: SEM 5/WEB TECH LAB (using java)/SESSION 2 24-08-2022/Calculator.java

package session_2;
import java.util.*;
public class Calculator {

    public static void main(String[] args) {
        while(true)
        {
            Scanner inp=new Scanner(System.in);
            System.out.print("Enter the choice - ");
            int ch=inp.nextInt();
            if(ch==111) break;
            System.out.print("Enter the numebers = ");
            int a = inp.nextInt();
            int b=inp.nextInt();
            
            switch(ch)
            {
                case 1:
                    System.out.println("Addition of a b = "+(a+b));
                    break;
                case 2:
                    System.out.println("Subtraction of a b = "+(a-b));
                    break;
                case 3:
                    System.out.println("Multiplication of a b = "+(a*b));
                    break;
                case 4:
                    System.out.println("Division of a b = "+(a/b));
                    break;
                case 5:
                    System.out.println("Square root of a and b = "+(Math.sqrt(a))+" and "+Math.sqrt(b));
                    break;
                case 6:
                    System.out.println("Power = "+Math.pow(a, b));
                    break;
                default:
                    System.out.println("Error input");
                    break;
            }
        }
    }
    
}
